cockatrice/src/gameselector.cpp

```diff
@@ -197,12 +197,20 @@ void GameSelector::actJoin()
 void GameSelector::retranslateUi()
 {
     setTitle(tr("Games"));
-    filterButton->setText(tr("&Filter games"));
-    clearFilterButton->setText(tr("C&lear filter"));
+    filterButton->setText(tr("Filter games"));
+    filterButton->setShortcut(QKeySequence("Ctrl+F"));
+    clearFilterButton->setText(tr("Clear filter"));
+    clearFilterButton->setShortcut(QKeySequence("Ctrl+D"));
     if (createButton)
-        createButton->setText(tr("C&reate"));
-    joinButton->setText(tr("&Join"));
+    {
+        createButton->setText(tr("Create"));
+        createButton->setShortcut(QKeySequence("Ctrl+N"));
+    }
+
+    joinButton->setText(tr("Join"));
+    joinButton->setShortcut(QKeySequence("Ctrl+J"));
```

Comment:
Turns out that is the Mac version of activated. The windows one is return. I would either leave it as Ctrl O or change the Ctrl J to the return key.